garnett/fields.py

```diff
@@ -75,6 +77,17 @@ def validate_translation_dict(all_ts):
             raise exceptions.ValidationError(f'Invalid value for language "{code}"')
 
 
+def translatable_default(
+    inner_default: Union[str, Callable[[], str]]
+) -> Dict[str, str]:
```

Comment:
Not really sure why this is on 3 lines, seems like it would be fine on 2

Reply:
I agree, but we are using the black autoformatter in garnett and thats just what it did

Reply:
Might look at changing it's line length some time though